src/ontology/mondo-edit.obo

```diff
@@ -432020,6 +432020,7 @@ subset: ordo_disease
 xref: NCIT:C181652 {source="MONDO:equivalentTo"}
 xref: Orphanet:529468 {source="MONDO:equivalentTo"}
 is_a: MONDO:0000001 {source="Orphanet:529468"} ! disease
```

Comment:
Should this remain as a parent?

Reply:
yeah, it's fine- you'll see this elsewhere too. It won't show up in the inferred version. If you look at Protege in the asserted hierarchy, you can see tons of classes under disease but these go away in the inferred version

<img width="580" alt="image" src="https://github.com/monarch-initiative/mondo/assets/6722114/cdaa1d38-1445-47f3-8030-847a20f0632b">


<img width="247" alt="image" src="https://github.com/monarch-initiative/mondo/assets/6722114/4f7f7b6e-db2f-483a-9028-266f4fe38ba2">